module/zfs/zvol.c

```diff
@@ -392,6 +392,8 @@ uzfs_ioc_stats(zfs_cmd_t *zc, nvlist_t *nvl)
 			    zv->checkpointed_time);
 			fnvlist_add_uint64(innvl, "zvol_workers",
 			    zv->main_zv->zvol_workers);
+			fnvlist_add_uint64(innvl, "quorum",
```

Comment:
how about using `fnvlist_add_uint8`?

Reply:
But internally we need to make use of `json_object_new_int64(val)` or `json_object_new_uint64(val)`. Because of making more changes just followed the same way how we are doing for "zvol_workers".

Reply:
i.e while converting we followed in below manner
https://github.com/openebs/cstor/blob/55b3edca2b173b1d444903772bc5ff214779a581/cmd/zfs/zfs_main.c#L7046

Reply:
ok